packages/redux-send-postMessage-thunk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "reduxSendPostMessageMiddleware",
```

Comment:
This isn't a legal NPM package name.  Per [npm conventions](https://docs.npmjs.com/files/package.json#name) package names *must* be lower case, and generally speaking are "-" separated.  Based on the names of the rest of the packages in this monorepo, I think this should be named `redux-send-postmessage` and the folder that contains it `redux-send-postMessage-thunk` should be renamed the same thing so that they match.

Reply:
Fixed.